pandas/tests/indexing/common.py

```diff
@@ -154,9 +154,12 @@ def get_value(self, f, i, values=False):
         # for a in reversed(i):
         #    v = v.__getitem__(a)
         # return v
-        with catch_warnings(record=True):
-            filterwarnings("ignore", "\\n.ix", FutureWarning)
-            return f.ix[i]
+
+        # TODO: this used to be f.ix[i]; is loc-then-iloc correct here?
```

Comment:
I think should just be `.iloc` looking at docstring?

Reply:
that was my first guess too, but that failed

Reply:
longer term, we should then probably clean up the tests using this get_value